configurations/manager/debian10.yaml

```diff
@@ -1,4 +1,4 @@
-ami_id_monitor: 'ami-074f8bbb689b1c1a0'  # Debian buster image - debian-10-amd64-20230601-1398, region - us-east-1
```

Comment:
I think I have better option for keep those up-to-date, I'll upload a PR soon with it.